src/core/meshcop/dtls.cpp

```diff
@@ -286,6 +291,13 @@ otError Dtls::Setup(bool aClient)
 
     OT_ASSERT(mCipherSuites[1] == 0);
     mbedtls_ssl_conf_ciphersuites(&mConf, mCipherSuites);
+    if (mCipherSuites[0] == MBEDTLS_TLS_ECJPAKE_WITH_AES_128_CCM_8)
+    {
+        mbedtls_ssl_conf_curves(&mConf, sCurves);
+#if defined(MBEDTLS_KEY_EXCHANGE__WITH_CERT__ENABLED)
```

Comment:
nit: the rest of this file uses `#ifdef`
```suggestion
#ifdef MBEDTLS_KEY_EXCHANGE__WITH_CERT__ENABLED
```